src/parser/parser.py

```diff
@@ -47,18 +112,64 @@ def count_articles(pdf_text):
             article_text += " " + pdf_text[i]
         i += 1
     for article in articles:
-        logging.debug("Article: " + str(article.number - 1) + " Text: " + article.text)
+        print("Article: " + str(article.number - 1) + " Text: " + article.text)
     return articles
 
 
-# Acts as main
+documents = {
+    "hash": "afafbfbdce8c40924edae00f6ce54f0c639ce42a2c0fbbfa6ab82ea6925827c51",  # Added one at last
+    "jurisdiction": "Monterrey",
+    "url": "http://www.guadalupe.gob.mx/wp-content/uploads/2019/09/Nuevo-Reglamento-Homologado-1.pdf",
+}
+document_list = []
+document_list.append(documents)
+
+# When the time comes to implement the DB, the commented stuff will allow that (with a few additions)
 def parse():
-    """Parses a PDF document
-    """
-    with open("regs.pdf", "rb") as pdf_file:
-        doc = slate.PDF(pdf_file)
-        final_text = ""
-        for page in doc:
-            final_text.join(page)
-        final_text = final_text.strip().lower().split()
-        count_articles(final_text)
+    """Parses all PDF documents that are in the DB"""
+    # docs = database.get_regulation_documents()  # DB Component
+    for document_to_parse in document_list:
+        # document = doc.to_dict()  # used when using DB Comp
+        retriever.get_document(
+            document_to_parse["url"]
+        )  # Change to document when using DB
+
+        # Check if the file is different, this has to be done in a different open file instance
+        # if not, Python returns some weird errors
+        hasher = hashlib.sha256()
+        with open("regs.pdf", "rb") as pdf_file:
+            file_buffer = pdf_file.read()
+            hasher.update(file_buffer)
+            if hasher.hexdigest() == document_to_parse["hash"]:
+                print("Downloaded document is the same as the one currently stored")
+                return "Succesful Operation - File did not change", 200
```

Comment:
Right, it is a success code [Mozilla has this page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200)

Reply:
Also I wanted to specify there that the file did not change, but it shouldn't make a difference vs a normal Successful Operation

Reply:
I meant to ask if returning ", 200" was a common practice

Reply:
As far as I know, yes. Can @ssmall and @anniefu chime in on this?

Reply:
To answer your question: it is more common to keep the business logic (parsing the document) separate from the application logic (determining what HTTP code to return). In this example, that would mean that the return code would be determined in `app.py` instead of here.

This way of keeping the different kinds of logic separate usually makes unit testing a lot easier

Reply:
I'll separate the logic for this, now, regarding Javier´s question, returning the 200 is fine?

Reply:
> regarding Javier´s question, returning the 200 is fine?

If I understand correctly what you're asking, you're asking if `200` is an appropriate HTTP status code to return in this case. Yes, I believe it is.

(That's not what I understood to be Javier's question, however)

Reply:
Yeah my question was definitely about determining the return code within app.py not about the definition of status 200

Reply:
I didn't understand at first then